server.js

```diff
@@ -27,6 +19,36 @@ app.get('/', (req, res) => {
   res.send('Hello world')
 })
 
+app.get('/genre/:genre', (req, res) => {
```

Comment:
I think it would be nice if this was part of the `/art` endpoint (and it'd be more RESTful) - then make use of query params to filter on genre. So you could have `/art` to return all art pieces and then `/art?genre= Surrealism`, etc.

In the RESTful world, we use the endpoints to describe the data returned, so for a `/genre` endpoint, you'd expect information about the genre, not pieces of art. if the endpoint was `/genres/:genre/art` then you'd expect art peices since the last part is 'art'. Does that make sense?